test-integration/main.js

```diff
@@ -27,6 +27,8 @@ const executeTestFunction = async (testFunctionObject, testContext) => {
 };
 
 module.exports = async (callback) => {
+    global.web3 = web3
```

Comment:
Why? Are you getting undefined?

Reply:
the `truffle-assertions` package requires a global instance of web3 but for some reason its not being recognized without doing this ☹️